_sections/40-body.md

```diff
@@ -79,7 +79,8 @@ The following attributes are supported on body elements. Note that most attribut
 | `jr:count`      | For the `<repeat>` element (see [repeats](#repeats)). This is one of the ways to specify how many repeats should be created by default.
 | `jr:noAddRemove`| For the `<repeat>` element (see [repeats](#repeats)). This indicates whether the user is allowed to add or remove repeats. Can have values `true()` and `false()`
 | `autoplay`      | For all form control elements, this automatically plays a [video or audio 'label'](#media) if the question is displayed on its own page, when the user reaches this page.
-| `accuracyThreshold` | For `<input>` with type `geopoint`, `geotrace`, or `geoshape` this sets the auto-accept threshold in meters for geopoint captures. [review]()
+| `accuracyThreshold` | For `<input>` with type `geopoint`, `geotrace`, or `geoshape`. Sets the auto-accept threshold in meters (decimals) for geopoint captures.
+| `unacceptableAccuracyThreshold` | For `<input>` with type `geopoint`, `geotrace`, or `geoshape`. Sets the threshold for unacceptable accuracy, in meters (decimals), for geopoint captures.
```

Comment:
@lognaturel do you think this is enough to close https://github.com/getodk/xforms-spec/issues/290?

Reply:
Yes! But both attributes only apply to `geopoint`, I think that's what the "review" note might have been about!